### ISSUE TITLE
Gleev adaptation for YPP

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#3322]
@dmtrjsg commented on [Tue Sep 27 2022](https://github.com/Joystream/marketing/issues/292)

## Dev Scope
![](https://user-images.githubusercontent.com/93924525/207915196-328a5aaa-7e85-45ae-aac0-03b58dc0f98e.png)

[Link to Figma> ](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1601%3A122582&amp;t=FMDH2UFYWNsPdW5o-1)

## Context
[Context of multiple gateways ](https://github.com/Joystream/marketing/issues/289) and rebranding of Atlas to Gleev will impact YPP project. This an issue raised addressing the overall change to the YPP project.

YPP incentive programme will only be carried out for Gleev App (which focusses on the more narrow set of categories, [listed here>](https://github.com/Joystream/atlas/issues/3273) , operated by JSG and we need to adapt the relevant parts of the YPP participants onboarding flow with regards to this.

There is also an impact on the category selection, which needs to be done as part of YPP onboarding flow.

:warning: :information_source: 
`YT sync scraps the YT category, and lets creator select a single category. What they select is default display category for the video category. We need to inform users that this will be propagated to all videos, but can be changed manually.'

>As specified in these issues:
>- https://github.com/Joystream/atlas/issues/3183
>- https://github.com/Joystream/youtube-synch/issues/42

## Scope
* [x] Add relevant copy to the Marketing website wrt Gleev YPP programme that would focus on Crypto/ Web3 Creators.
* [x] Add the criteria for YPP partnership onboarding flow to the assessment criteria-> relevant content selection based on categories. List of categories is provided in the additional info section below
* [ ] YPP dashboard needs to be updated for posting YT content to Joystream channel only adhering to the content strategy of Gleev (Web3).
* [x] #3509

:new: out of scope: ⛔ 

(since we do not have a dedicated page for Apps now)

* [ ] Mention that Gleev will only focus on Web3 creators on the page which contains multiple apps (covered in scope of 
* #289)
* [ ] Mention YPP programme on the Gleev app (label and tooltip, or smth else)

#### Additional Info :information_source: 
Categories: (Web3/ Crypto/ Blockchain)
Investing
Podcasts
Conferences
Reviews
Tutorials
News
Memes
Interviews
Analysis
Predictions
Trading

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202078489349362/1203673795150481) by [Unito](https://www.unito.io)

**Comment by toiletgranny** [comment]
@dmtrjsg As discussed before, since https://github.com/Joystream/marketing/issues/297 is ready to tackle, I'm putting this task on hold. Below I include a changelog outlining what I have managed to accomplish so far. Please note that these changes are ready to be implemented. The below changelog, along with designs and docs, [can also be found in this Figma file](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1818%3A91087&t=6qLLqdKGWEN1GZ9A-4).

> 1. Updated all illustrations (across both the landing page and the app) to feature Gleev logo, name and sample content,
> 2. Replaced all instances of the “Atlas” name in copy with “Gleev”,
> 3. Updated the Authorized channel dialog to replace Atlas logo with Gleev [(link)](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1815%3A620007),
> 4. Updated the YT channel card component to replace Altas logo with Gleev [(link)](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1816%3A73109).

**Comment by dmtrjsg** [comment]
Edits to the original scope: 

<img width="676" alt="Screenshot 2022-12-13 at 15 30 22" src="https://user-images.githubusercontent.com/93924525/207306763-5185e3e0-3250-4913-9311-813b2ad817bd.png">

**Comment by toiletgranny** [comment]
@dmtrjsg This task is ready to be reviewed and/or implemented based on the designs, documentation, and changelog that [can be found in this Figma file](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=3338%3A84773&t=Hy0I075Taa1HEsTU-4). 🚀 

---

> ## Scope
> [...]
> * [ ]  YPP dashboard needs to be updated for posting YT content to Joystream channel only adhering to the content strategy of Gleev (Web3).

I haven't addressed the above item from the scope as the Figma designs had never accurately reflected the detailed program mechanics anyway. Since we haven't received any documentation for the exact requirements and the list of steps required for a given task to be considered complete, the copy in Figma is just @KubaMikolajczyk's best guess he had at the time of designing this.  

This touches upon this [unresolved discussion](https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5?node-id=1789:116338#257316000) from Figma.

![CleanShot 2022-12-13 at 13 36 39](https://user-images.githubusercontent.com/35307309/207319957-3134b3f2-cff2-4608-a0a2-8bb323f683f6.png)

Please, let me know if anything has changed since then. If the program mechanics are documented somewhere, I'm happy to update Figma designs to reflect that.

**Comment by dmtrjsg** [comment]
@toiletgranny thank you for handling this. The programme details and the specific steps I will make sure to populate 👌

**Comment by dmtrjsg** [comment]
Final copy will be updated in a separate task, along with parameters. Captured here (and blocked by operating model  sim validation): 
- https://github.com/Joystream/atlas/issues/3570

@WRadoslaw fyi
